parser/mysql/const.go

```diff
@@ -155,6 +155,9 @@ const (
 	ClientPluginAuth
 	ClientConnectAtts
 	ClientPluginAuthLenencClientData
+	ClientCanHandleExpiredPasswords
+	ClientSessionTrack
```

Comment:
Just a suggestion, but in https://github.com/pingcap/tidb/blob/b81cb5a9d78cbb832a6763d9bbe1a3fba92df513/server/conn.go#L1311-L1380 we have comments for the parts of the protocol not supported. Could we add it for the consts too?

```suggestion
	ClientCanHandleExpiredPasswords // Not supported: https://github.com/pingcap/tidb/issues/9709
	ClientSessionTrack // Not supported/applicable: https://dev.mysql.com/worklog/task/?id=6128
```

This might make it easier for the next person debugging.